src/tasks/leveling.ts

```diff
@@ -790,11 +787,9 @@ export const LevelingQuest: Quest = {
       completed: () => getKramcoWandererChance() < 1.0 || !have($item`Kramco Sausage-o-Matic™`),
       do: $location`Noob Cave`,
       outfit: {
+        ...baseOutfit(),
```

Comment:
Anywhere you call baseOutfit() directly has to be part of a lambda function like on line 764.